src/cpp/src/image_generation/flux_pipeline.hpp

```diff
@@ -415,20 +442,15 @@ class FluxPipeline : public DiffusionPipeline {
 
         m_generation_config = ImageGenerationConfig();
 
-        if (m_pipeline_type != PipelineType::IMAGE_2_IMAGE) {
-            m_generation_config.height = transformer_config.m_default_sample_size * vae_scale_factor;
-            m_generation_config.width = transformer_config.m_default_sample_size * vae_scale_factor;
-        }
```

Comment:
I suppose we need to keep code which computes default image sizes, with removed condition about image2image

Other pipelines provides default sizes